src/Reflection/ReflectionProperty.php

```diff
@@ -375,7 +378,9 @@ public function setValue(mixed $object, mixed $value = null): void
         if ($this->isStatic()) {
             $this->assertClassExist($implementingClassName);
 
+            /** @psalm-suppress UnusedClosureParam */
```

Comment:
Naming a parameter with an underscore prefixing the name works too

Reply:
Modified.